web/src/containers/viewOnly/activities/Activity.js

```diff
@@ -27,7 +27,7 @@ const Activity = ({ activity, activityIndex }) => {
               )
             )}
           </ul>
-        </p>
+        </div>
```

Comment:
Nesting a `<ul>` within. `<p>` tag is frowned upon, apparently. Or, at least that's what the JS console says.